webpack.config.js

```diff
@@ -46,9 +46,12 @@ module.exports = {
           {
             test: /\.(ts|tsx|jsx|js)$/,
             loader: require.resolve('ts-loader'),
+            options: {
+              transpileOnly: true,
+            }
```

Comment:
I saw my previous comments that suggested to add it but right now I see no reason, please remove.

Reply:
removed